### ISSUE TITLE
symfony var-dumper: possiblitiy to copy the dumped data as json or php array 

### ISSUE THREAD

**Issue by nuryagdym** [symfony/symfony#59249]
### Description

Would be possible to add buttons to the dump()/dd() results so that we can copy the response as json or php array?
My use case is I want the data for creating tests. 
I could json encode myself, but I often forget to json encode and reproducing the response is sometimes takes a time.
Also, json encode does not give me php array.

**Comment by stof** [comment]
What would you expect as PHP array when you dump an object ?

The goal of VarDumper is not to export code re-instantiating an object (VarExporter solves this use case) but to be a debugging tool showing the content of a value. It is a better version of `var_dump` (which does not produce PHP code either)

**Comment by nuryagdym** [comment]
Yeah, for objects it may be an issue. But for arrays and other primitive PHP types it maybe applicable.

I did not know about VarExporter component. I will check that out and let you know (hopefully will not forget) if it meets my requirements

**Comment by nicolas-grekas** [comment]
For tests, you should use the VarDumperTestTrait. It's built for this use case.

**Comment by nuryagdym** [comment]
@nicolas-grekas thanks for the suggestion, but I did not meant that use case.
For example, I do request on browser and add temporary `dump()/dd()` line to see the data (sometimes I am too lazy to use xdebug). And copy the dump data to create unit test.

**Comment by stof** [comment]
@nuryagdym if your code is to have PHP code performing the initialization, what you need is not VarDumper, but VarExporter. You need a better `var_export`, not a better `var_dump`
